### PR TITLE
docs: Support varbinary in tables endpoint

### PR DESCRIPTION
**Pull request by helanto**
Screenshot 2024-04-17 at 11 11 56 AM
Screenshot 2024-04-17 at 11 12 28 AM

### CODE REVIEW

File: api-reference/tables/endpoint/insert.mdx [hunk 1]
```diff
@@ -22,6 +22,11 @@ CSV files must use a comma as delimiter, and the column names in the header must
 These are files where each line is a complete JSON object which creates one table row.
 Each line must have keys that match the column names of the table.
 
+## Data types
+DuneSQL supports a variety of types which are not natively supported in many data exchange formats. Here we provide guidance on how to work with such types.
+### Varbinary values
+When uploading varbinary data using JSON or CSV formats, you need to convert the binary data into its hexadecimal textual representation. Specifically, input data should contain an even number of characters in the range `[0-9a-fA-F]` always prefixed with `0x`.
```
Comment (waddahAldrobi) [inline api-reference/tables/endpoint/insert.mdx:28]: Can you add an example, might not make sense to a lot of people :)

**Review by waddahAldrobi — APPROVED** [review]

**Review by dsalv — APPROVED** [review]
Nice!

**Comment by vegarsti** [comment]
Nice!

**Review by hawkaa — COMMENTED** [review]
We did discuss supporting base64 too. We never did it?

[PRO-860 : Convert `0x` hex strings and base64 strings to binary](https://linear.app/dune/issue/PRO-860/convert-0x-hex-strings-and-base64-strings-to-binary)

**Comment by hawkaa** [comment]
> > We did discuss supporting base64 too. We never did it?
> > [PRO-860 : Convert `0x` hex strings and base64 strings to binary](https://linear.app/dune/issue/PRO-860/convert-0x-hex-strings-and-base64-strings-to-binary)
> 
> We added support for `base64` as well

Right. Is the documentation accurate for base64? Reading the text + seeing the example makes me think we only support `0x` strings.

**Comment by helanto** [comment]
> > > We did discuss supporting base64 too. We never did it?
> > > [PRO-860 : Convert `0x` hex strings and base64 strings to binary](https://linear.app/dune/issue/PRO-860/convert-0x-hex-strings-and-base64-strings-to-binary)
> > 
> > 
> > We added support for `base64` as well
> 
> Right. Is the documentation accurate for base64? Reading the text + seeing the example makes me think we only support `0x` strings.

It's here 👉 #140

**Pull request merged**